MultipleFileUpload/MultipleFileUpload.php

```diff
@@ -186,8 +189,9 @@ public static function cleanCache()
 	 */
 	public static function getQueuesModel()
 	{
-		if (!self::$queuesModel) { // if nothing is set, setup sqlite model, which should work on all systems with SQLite
-			self::setQueuesModel(new Model\SQLite3\Queues());
+		if (!self::$queuesModel) { // if nothing is set, setup sqlite model, which should work on all systems with MySQL
+            $queuesModel = new Model\MySQL\Queues(self::$container, self::$container->getService("database.default"));
+			self::setQueuesModel($queuesModel);
```

Comment:
model inject as string namespace in constructor as in Registrator->getInterfaces() ?